docs/user-guides/Git_branching_strategy.md

```diff
@@ -0,0 +1,77 @@
+# Guide: Git Branching Strategy
+
+- [Guide: Git Branching Strategy](#guide-git-branching-strategy)
+  - [Git branching strategy diagram](#git-branching-strategy-diagram)
+  - [Branch naming conventions](#branch-naming-conventions)
+  - [To start work on a new ticket](#to-start-work-on-a-new-ticket)
+  - [To test a ticket](#to-test-a-ticket)
+  - [Identify release candidates](#identify-release-candidates)
+  - [Once release candidates are merged to main](#once-release-candidates-are-merged-to-main)
+  - [Hotfix process](#hotfix-process)
+  - [Rebase](#rebase)
+  - [References](#references)
+
+## Git branching strategy diagram
+
+![Git branching strategy diagram](../diagrams/Git_branching_strategy.png)
+
+## Branch naming conventions
+
+- We prefer task over 'feature' as task better encapsulates the short-lived nature of transient branches.
+- It is a task to be delivered (on the way to delivering a full feature) rather than the full feature.
+- Consequently temporary branches created from main should follow this naming pattern: `task/JIRA_REF_Min_three_word_desc`
+  - Where the minimum three word description starts with an initial capital
+  - And the max length for a branch name is 60 characters.
+
+- Valid branch names would be
+  - `task/DR_1_Update_s3_terraform`
+  - `task/DR_9_Add_new_data_item`
```

Comment:
I'd personally rather keep this as the actual Jira format - i.e. `task/DR-9_Add_new_data_item`.
This makes it much easier to pull the Jira reference out of the branch name using a regex, and stops you having to do some odd formatting with it once you pull it out.

Reply:
As a mildly controversal comment - do we really care about the last bit of the branch name? Branches come and go, commits last forever.

Reply:
I think the dev agreement was that the branching name would all be '_' re DR-1211 as having the mixture adds confusion.

Reply:
I have coded around the underscored jira ref in the branch name - I don't think that is much of an issue given team's preference for consistency around use of underscores